pkg/sfu/buffer/buffer.go

```diff
@@ -33,13 +33,14 @@ type pendingPacket struct {
 }
 
 type ExtPacket struct {
-	Arrival              int64
-	Packet               *rtp.Packet
-	Payload              interface{}
-	KeyFrame             bool
-	RawPacket            []byte
-	SpatialLayer         int32
-	TemporalLayer        int32
+	VideoLayer
+	Arrival   int64
+	Packet    *rtp.Packet
+	Payload   interface{}
+	KeyFrame  bool
+	RawPacket []byte
+	// Spatial              int32
```

Comment:
Can remove commented out stuff. Sorry, I keep bugging people about it. I really like clean stuff or at least a comment as to why something is commented out so that somebody reading this 6 months from now has some idea.